config.yml

```diff
@@ -0,0 +1,21 @@
+name: mid-west-coast-AZ-NV
+start_date: 2020-01-31
+end_date: 2020-10-01
+dt: 0.25
+
+spatial_setup:
+  base_path: data/west-coast-AZ-NV
+
+  setup_name: west-coast-AZ-NV
+  geodata: geodata.csv
+  mobility: mobility.txt
+  popnodes: pop2010
+
+interventions:
+  base_path: COVIDScenarioPipeline/data
+
+  NoNPI: NPI_Scenario1_None.R
+  SC: NPI_Scenario2_School_Closure.R
+  BI1918: NPI_Scenario3_Bootsma_1918Influenza.R
+  KansasCity: NPI_Scenario4_KansasCity.R
+  Wuhan: NPI_Scenario5_Wuhan.R
```

Comment:
plan is to have these be dynamically defined (by globbing a filepath) -- will do in future (not backwards compatible)

Reply:
oh great idea